### PR TITLE
You can only observe living mobs

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Title. 
## Why It's Good For The Game

Because being able to observe people in lobby and click buttons for them is funny.
## Changelog
:cl:
fix: You can only observe living mobs
/:cl:

### CODE REVIEW

**Comment by DeltaFire15** [comment]
This is not the only vector of this, you can also observe via enabling "autoobserve" in the orbitting tgui, then observing a dead (lobbied) player.

**Comment by ivanmixo** [comment]
The second one.

**Review by TiviPlus — APPROVED** [review]

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -792,15 +792,15 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
...
+	var/mob/target = tgui_input_list(usr, "Please select a mob:", "Observe", GLOB.mob_living_list)
 	if(!target)
 		return
 
 	do_observe(target)
 
 ///makes the ghost see the target hud and sets the eye at the target.
 /mob/dead/observer/proc/do_observe(mob/target)
-	if(!client || !target || !istype(target))
+	if(!client || !target || !istype(target) || !(target in GLOB.mob_living_list))
```
Comment (TiviPlus) [inline code/modules/mob/dead/observer/observer.dm:803]:
```suggestion
	if(!client || !target || !isliving(target))
```
cant u do this actually

Comment (ivanmixo) [reply code/modules/mob/dead/observer/observer.dm:803]: True

**Pull request merged**